.ci/build-push-bundle.sh

```diff
@@ -24,14 +25,22 @@ docker build -t byoh-bundle .
 docker rm -f byoh-bundle-container
 
 echo "executing docker image"
-docker run -e BUILD_ONLY -e CONTAINERD_VERSION -e KUBERNETES_VERSION -e KUBERNETES_MAJOR_VERSION -e ARCH --name byoh-bundle-container -i byoh-bundle /bin/bash
+docker run -e BUILD_ONLY -e CONTAINERD_VERSION -e KUBERNETES_VERSION -e KUBERNETES_MAJOR_VERSION -e ARCH -e UBUNTU_VERSION --name byoh-bundle-container -i byoh-bundle /bin/bash
 
 echo "creating bundle dir to push k8s packages"
 mkdir -p ./bundle
 
 echo "coping bundle from docker image"
 docker cp byoh-bundle-container:/bundle/. ./bundle/
 
-echo "pushing oci bundle to quay.io/platform9/byoh-bundle-ubuntu_20.04.1_x86-64_k8s"
-./imgpkg push -f ./bundle -i quay.io/platform9/byoh-bundle-ubuntu_20.04.1_x86-64_k8s:$BUNDLE_VERSION
+# Determine bundle name based on Ubuntu version
 
+if [ "$UBUNTU_VERSION" = "20.04" ]; then
+    BUNDLE_NAME="byoh-bundle-ubuntu_20.04.1_x86-64_k8s"
+else
```

Comment:
```suggestion
elif [ "$UBUNTU_VERSION" = "22.04" ];
```
We should check for Ubuntu 22.04 specifically - the else part can raise an error saying version x is not supported.

Reply:
Bito has the exact same suggestion